api/lib/api/performance_manager.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Api.PerformanceManager do
+  @moduledoc """
+  The PerformanceManager context.
+  """
+
+  import Ecto.Query, warn: false
+  alias Api.Repo
+
+  alias Api.PerformanceManager.MetadataLog
+
+  @doc """
+  Returns the list of metadata_log.
+
+  ## Examples
+
+      iex> list_metadata_log()
+      [%MetadataLog{}, ...]
+
+  """
+  def list_metadata_log do
+    Repo.all(MetadataLog)
+  end
+
+  @doc """
+  Gets a single metadata_log.
+
+  Raises `Ecto.NoResultsError` if the Metadata log does not exist.
+
+  ## Examples
+
+      iex> get_metadata_log!(123)
+      %MetadataLog{}
+
+      iex> get_metadata_log!(456)
+      ** (Ecto.NoResultsError)
+
+  """
+  def get_metadata_log!(pk_id), do: Repo.get!(MetadataLog, pk_id)
```

Comment:
only functions here right now are to get all of the metadata log table or a just a single row from it.

my understanding is that this section gets filled out quiet a bit more once we bring in the other tables. potentially, this is the place where more complex join queries are also stored?